test/Microsoft.OpenApi.Readers.Tests/Resources.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.IO;
+
+namespace Microsoft.OpenApi.Readers.Tests
+{
+    internal static class Resources
+    {
+        /// <summary>
+        /// Get the file contents.
+        /// </summary>
+        /// <param name="fileName">The file name with relative path. "/V3Tests/Samples/OpenApiCallback/..</param>
```

Comment:
>" [](start = 69, length = 1)

nit, Either remove this quotation mark or add one at the end.

Same thing with the comment below.